tests/test_tap_mysql.py

```diff
@@ -74,7 +74,6 @@ def setUpClass(cls):
     def test_decimal(self):
         self.assertEqual(self.schema.properties['c_decimal'],
                          Schema(['null', 'number'],
-                                sqlDatatype='decimal(10,0)',
```

Comment:
It makes sense to remove the sqlDatatype from these tests, but I think we should add them to a different test somewhere else. That's a good bit of functionality that I think is good to have tests for.